webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js

```diff
@@ -143,6 +143,9 @@ class SakaiRubricsManager extends RubricsElement {
 
   sortRubrics(event) {
     const sortInput = event.target.className;
+    if (sortInput === "") {
+      return;
+    }
     const sortInputValue = sortInput.toString().toLowerCase();
     const rubric = sortInputValue.split('-');
```

Comment:
I'd just go:
const rubricClass = sortInput && sortInput.toLowerCase().split("-")[0];
You don't even need the falsy test, then.